plugin/src/main/java/org/opensearch/ml/sdkclient/RemoteClusterIndicesClient.java

```diff
@@ -73,17 +68,19 @@
 @Log4j2
 public class RemoteClusterIndicesClient implements SdkClient {
 
-    private OpenSearchClient openSearchClient;
-
     @SuppressWarnings("unchecked")
     private static final Class<Map<String, Object>> MAP_DOCTYPE = (Class<Map<String, Object>>) (Class<?>) Map.class;
 
+    private OpenSearchClient openSearchClient;
+    private JsonpMapper mapper;
+
     /**
      * Instantiate this object with an OpenSearch Java client.
      * @param openSearchClient The client to wrap
      */
     public RemoteClusterIndicesClient(OpenSearchClient openSearchClient) {
         this.openSearchClient = openSearchClient;
+        this.mapper = openSearchClient._transport().jsonpMapper();
```

Comment:
is `_trasport()` internal method? JsonMapper should rather be passed in as part of constructor

Reply:
It's a `public` method on the `ApiClient` class:

https://github.com/opensearch-project/opensearch-java/blob/3af2d776903123c336686ffbfaafc5503f949b5b/java-client/src/main/java/org/opensearch/client/ApiClient.java#L66-L68

It's specifically listed exactly like this in the opensearch-java documentation:
https://github.com/opensearch-project/opensearch-java/blob/3af2d776903123c336686ffbfaafc5503f949b5b/guides/generic.md?plain=1#L81